app/src/main/java/com/raghav/spacedawn/ui/viewmodels/BaseViewModel.kt

```diff
@@ -0,0 +1,8 @@
+package com.raghav.spacedawn.ui.viewmodels
+
+import androidx.lifecycle.ViewModel
+import kotlinx.coroutines.Job
+
+open class BaseViewModel : ViewModel() {
```

Comment:
I would prefer not to have this in a separate base class this can be included in the ViewModel itself since its not that big of a code to share on small scale this would affect the readability more than it would help in scailing

Reply:
Using base classes to use redundant code is generally a good idea though with a drawback that as many classes inherit the same class refactoring becomes difficult. I can remove that if it's that important.

Reply:
please remove this as here drawbacks outweigh the benefits

Reply:
ok